src/main/scala/nl.knaw.dans.easy.bag2deposit/collections/Collection.scala

```diff
@@ -49,10 +55,35 @@ object Collections extends DebugEnhancedLogging {
       .tried
   }
 
-  private val skosCsvFormat = RFC4180.withHeader("URI", "prefLabel", "definition", "broader", "notation")
-  private val collectionCsvFormat = RFC4180.withHeader("name", "ids", "type", "comment")
+  private val collectionCsvFormat = RFC4180
+    .withHeader("name", "EASY-dataset-id", "type", "comment", "members")
+    .withDelimiter(',')
+    .withRecordSeparator('\n')
+    .withAutoFlush(true)
+
+  private def parseCollectionRecord(r: CSVRecord) = Try {
+    val memberIds = Try(r.get("members"))
+      .map(_.split(", *"))
+      .getOrElse(Array[String]())
+    Collection(
+      r.get("name"),
+      r.get("EASY-dataset-id").split(", *").filter(_.startsWith("easy-dataset:")),
+      Try(r.get("type")).getOrElse(""), // TODO assuming IllegalArgumentException
+      Try(r.get("comment")).getOrElse(""),
+      memberIds, // compiler error when inlined
+    )
+  }.getOrElse(throw new IllegalArgumentException(s"invalid line $r"))
+
+  private def writeCollectionRecord(printer: CSVPrinter, c: Collection): Try[Collection] = Try {
```

Comment:
@jo-pol Looks like you are putting multiple values in a cell. I hope I am wrong, or that you know what you are doing...

Reply:
Indeed I know what I'm doing. These fields are split on `,` before processing.